src/main/java/com/example/Customer.java

```diff
@@ -0,0 +1,9 @@
+package com.example;
+
```

Comment:
Here should be Javadoc. Every `public`, `protected`, and package-private thing in a code _must_ be documented. When you show your code to colleagues it _needs_ documentation. It's written and maintained together with the executable code. It _is_ an integral part of the code.